client-common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -125,6 +125,7 @@ public final class ApiEndpointsV1
     public static final String PER_OPERATIONAL_JOB = OPERATIONAL_JOBS + '/' + OPERATIONAL_JOB_ID_PATH_PARAM;
     public static final String LIST_OPERATIONAL_JOBS_ROUTE = API_V1 + CASSANDRA + OPERATIONAL_JOBS;
     public static final String OPERATIONAL_JOB_ROUTE = API_V1 + CASSANDRA + PER_OPERATIONAL_JOB;
+    public static final String STREAM_STATS_ROUTE = API_V1 + CASSANDRA + "/stats/stream";
```

Comment:
Should be plural here if you are retrieving all the streams
```suggestion
    public static final String STREAM_STATS_ROUTE = API_V1 + CASSANDRA + "/stats/streams";
```